docs/comparison.md

```diff
@@ -0,0 +1,28 @@
+# Model Comparison
+
+The `jetfuelburn` package implements multiple fuel burn calculation models, each with different levels of complexity, data requirements, and accuracy. This page provides a comprehensive comparison to help you choose the most appropriate model for your use case.
+
+## Aircraft Type Coverage by Model
+
+The following table shows which aircraft types are supported by each model in the `jetfuelburn` package:
+
+|                                                               |A220-300 |A320neo  |A330-300 |A340-300 |B777-300ER| Comment   |
+|---------------------------------------------------------------|---------|--------|--------|--------|----------|------------------------------------------|
+| **[US DOT Statistics (2024)][jetfuelburn.statistics.usdot]**  |✓        |✓        |✓        |✗        |✓          |                                           |
```

Comment:
Hyperlink to functions using this `mkdocs` syntax [text here][python.module.path.here].